packages/connect-popup/src/index.tsx

```diff
@@ -50,6 +50,13 @@ const handleMessage = (event: MessageEvent<PopupEvent | UiEvent>) => {
         return;
     }
 
+    if (data.type === POPUP.SHOW_ERROR) {
```

Comment:
is there any specific reason why this code block is here (line 53-58) and not part of switch clause on line 95?